sdk/template/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# Lines starting with '#' are comments.
```

Comment:
What is this file for?

Reply:
The template release fails when CODEOWNERS is missing, because some of our docs scripts use it to look for that data. Apparently we haven't actually released template in a while, so I ran into this issue when trying to validate all the functionality.

Reply:
I think that is just a path issue we have never had CODEOWNER files directly in service directories. The only one should be under the .github folder. This file should not be needed.

Reply:
I assume you still plan to delete this file.